src/kibana/plugins/discover/components/field_chooser/field_chooser.js

```diff
@@ -173,7 +174,7 @@ define(function (require) {
               metric: [{
```

Comment:
I think that 174-179 can be deleted. Looks like old code that wasn't cleaned up.